README.md

```diff
@@ -147,6 +201,14 @@ There are two types of configuration that `wrangler` uses: global user and per p
 
 1. Publish your project:
 
+    To publish to a workers.dev subdomain:
     ```
     wrangler publish
     ```
+
+    To publish to a domain you have registered with Cloudflare, add a `route` and a `zone_id` to your
+    `wrangler.toml`. Then run:
+
+    ```
+    wrangler publish --release
```

Comment:
OK, neat – so my understanding of this update means that wrangler defaults to a `workers.dev` subdomain deploy, right? To do a zone deploy, you can use `wrangler publish --release`?

Reply:
Is it possible to simplify this from a usability standpoint a bit based on what we know from `wrangler.toml`? If a `zone_id` is present, can we add the `--release` flag for the user so they don't have to work out which one to run themselves?

(If I'm totally misunderstanding the entire differentiation between these commands, let me know!)